### DISCUSSION TITLE
New HIP backend?

### DISCUSSION THREAD

**Discussion by lenhone** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#465] [General]
Probably to early to ask but with core 2.2.11 being supported in plugin version  3.3.21 can the [HIP back end](https://gpuopen.com/learn/introducing-prorender-sdk-2-02-11/) be tested?

Observations:
- Setting "CREATION_FLAGS_ENABLE_HIP" in context.py produces renders... not sure if it does anything though apart from render a little slower.
- Running Blender with environment variable "TH_FORCE_HIP=true" crashes the GPU requiring a reset.

**Comment by bsavery** [comment]
We haven't tested it yet in the plugin.  So probably best to wait.  The support was just added, and we don't have a build with 2.2.11 SDK up yet.  Unless you're building yourself.

**Reply by lenhone** [reply]
No worries will wait for official release, thanks.
